includes/class-wcpdf-main.php

```diff
@@ -845,37 +847,68 @@ public function set_header_logo_height( $document_type, $document = null ) {
 			<?php
 		}
 	}
-
+	
 	/**
-	 * Remove attachments older than 1 week (daily, hooked into wp_scheduled_delete )
+	 * Remove temporary files from paths older than 1 week (daily, hooked into wp_scheduled_delete ) or manually
 	 */
-	public function attachments_cleanup() {
-		if ( !function_exists("glob") || !function_exists('filemtime') ) {
-			// glob is required
-			return;
-		}
-
-		
-		if ( !isset( WPO_WCPDF()->settings->debug_settings['enable_cleanup'] ) ) {
-			return;
-		}
+	public function temporary_files_cleanup( $now = false ) {
+		global $wp_filesystem;
 
+		require_once( ABSPATH . 'wp-admin/includes/file.php' ); // required for WP_Filesystem()
 
-		$cleanup_age_days = isset(WPO_WCPDF()->settings->debug_settings['cleanup_days']) ? floatval(WPO_WCPDF()->settings->debug_settings['cleanup_days']) : 7.0;
-		$delete_timestamp = time() - ( intval ( DAY_IN_SECONDS * $cleanup_age_days ) );
+		// loads WP_Filesystem()
+		if ( ! WP_Filesystem() ) {
+			WP_Filesystem();
+		}
```

Comment:
this is strange, why do you need to call this twice? shouldn't `WP_Filesystem()` have the same effect? Or in other words, why would it return false at the first call and then succeed at the second?